plugins/woocommerce/src/Internal/DataStores/Orders/DataSynchronizer.php

```diff
@@ -152,41 +152,49 @@ public function get_sync_status() {
 	public function get_current_orders_pending_sync_count(): int {
 		global $wpdb;
 
-		// TODO: Remove the usage of the fake pending orders count once development of the feature is complete.
-		$count = get_option( self::FAKE_ORDERS_PENDING_SYNC_COUNT_OPTION );
-		if ( false !== $count ) {
-			return (int) $count;
-		}
-
-		$orders_table = $wpdb->prefix . 'wc_orders';
+		$orders_table                = $wpdb->prefix . 'wc_orders';
```

Comment:
Unless there's a good reason to hardcode, maybe we can do this instead:

```suggestion
		$orders_table                = $this->data_store->get_orders_table_name();
```

Reply:
addressed in https://github.com/woocommerce/woocommerce/pull/34167/commits/beda0bc425abd7d9fc6220e9f4a3929cf4d7d485